docs/apple-privacy-manifest.md

```diff
@@ -0,0 +1,431 @@
+
+* It’s your responsibility to check the accuracy of the privacy manifest within the .NET MAUI app and if any third-party components included in your .NET MAUI project require any declarations in you privacy manifest. It’s recommended that you search these third-party components for any references to a privacy manifest declaration.
+
+* If you’re developing an application using .NET MAUI as a library, check if your native application code collects any of the following information outside of the .NET MAUI project:
+    * [Data collection](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_data_use_in_privacy_manifests)
+    * [Required Reasons APIs][RequiredReasonAPI]
+
+  If it does, then you must declare their usage in the privacy manifest file.
+
+## Privacy manifest for Binding projects
+If you are Binding project owner, and you provide a NuGet package of that binding project to be included in a .NET for iOS/macOS or .NET MAUI app then you will need to provide documentation for the consumers of your package in order for them to be able to create the `PrivacyInfo.xcprivacy` file properly. It is currently not possible for Binding project authors to include a `PrivacyInfo.xcprivacy` file that will be recognized by Apple when submitting an app.
+
+## C# .NET APIs in .NET MAUI
+There are three layers that could potentially use any of the [Required Reasons APIs][RequiredReasonAPI]:
+
+* .NET runtime and Base Class libraries (BCL)
+* .NET for iOS SDK
+* .NET MAUI SDK
+
+The .NET runtime and BCL use and ship as part of your app bundle APIs in the following categories:
+* [File timestamp APIs][FileTimestampAPIs]
+* [System boot time APIs][SystemBootTimeAPIs]
+* [Disk space APIs][DiskSpaceAPIs]
+
+Therefore, all .NET applications must include the above categories in `PrivacyInfo.xcprivacy` file. See the [example](#an-example) for the format and reason codes needed.
+
+.NET for iOS and .NET MAUI build on top of the .NET runtime and BCL and include additional API usages as defined in the following tables. The tables provide lists of C# .NET APIs that call the [Required Reasons APIs][RequiredReasonAPI] organized by category. If your application, SDK or package code calls any of the APIs from these lists, declare the reasons for their use in your privacy manifest file following the guidelines specified in Apple’s documentation on [Required Reasons APIs][RequiredReasonAPI].
+
+**Note:** The following lists are verified only for .NET MAUI versions 8.0.0 and later.
+
+### [File timestamp APIs][FileTimestampAPIs]
+
+The following APIs either directly or indirectly access file timestamps and require reasons for use. Use the string `NSPrivacyAccessedAPICategoryFileTimestamp` as the value for the `NSPrivacyAccessedAPIType` key in your `NSPrivacyAccessedAPITypes` dictionary. Refer to [File timestamp APIs][FileTimestampAPIs] for any additional relevent values to add to the `NSPrivacyAccessedAPITypeReasons` array.
```

Comment:
What about BCL APIs? Will they be listed separately?

Reply:
I have them for .NET but not exactly for mono. I left them out since the API usages would be present in the runtime and BCL libraries. Assuming that Apple would be doing static analysis of the app and not runtime the category and reason would always be required regardless of API use.

Reply:
I'm not sure I understand; from Apple's perspective it doesn't matter if the API usage is through the BCL or our bindings for Foundation/UIKit/AppKit.

Reply:
Yeah I agree here with @rolfbjarne, I think we should also be listing the BCL types here as Apple will not care where it is coming from

Reply:
Ok, they have been documented.